### DISCUSSION TITLE
failed to submit a bug

### DISCUSSION THREAD

**Discussion by xxjoe2** [bromite/bromite#1537] [General]
sorry for being noob

i have submitted a bug [here](https://github.com/bromite/bromite/issues/1535) but it was marked as closed and i got 

Thanks for submitting this issue!
I noticed that it is missing the template, please edit the issue to match the template......

what should i do actually? can i remove it? should i resubmit it?

please suggest

thanks a lot

**Comment by csagan5** [comment]
It says to edit the issue to match the template, so that is what you should actually do.

You replied "n/a" to the question "Can the bug be reproduced with corresponding Chromium version?"; you have to test it with Chromium before posting a bug report. The template shown when you submit the issue explains everything:

> Please pick the same version of Chromium as Bromite from here: https://github.com/bromite/chromium/releases
> If the bug is reproducible then it might be a configuration issue or an upstream bug. Upstream bugs can be reported on the [Chromium issue tracker](https://bugs.chromium.org/p/chromium/issues/list) and do not forget to read [Chromium project bug reporting guidelines](https://www.chromium.org/for-testers/bug-reporting-guidelines) first.
> 
> If the bug is related to functionality that does not exist in Chromium then answer "No".

**Reply by xxjoe2** [reply]
alright. so i edit the post again will do or should i open a new one?

**Reply by xxjoe2** [reply]
i have edited the old one and submitted a new one [here](https://github.com/bromite/bromite/issues/1539) but both didnt work
